arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -728,7 +727,7 @@
         "tags": [
           "Queues"
         ],
-        "operationId": "Queues_CreateOrUpdateAuthorizationRule",
+        "operationId": "Queue_CreateOrUpdateAuthorizationRule",
```

Comment:
In the operationId: Queue**s** is good. We use plural form for the operation group. Why was this changed to singular form?